calico_node/startup/startup.go

```diff
@@ -633,6 +636,20 @@ func checkConflictingNodes(client *client.Client, node *api.Node) {
 	}
 }
 
+// Checks that the filesystem is as expected and fix it if possible
+func ensureFilesystemAsExpected() {
+	runDir := "/var/run/calico"
+	// Check if directory already exists
+	if _, err := os.Stat(runDir); err != nil {
+		// Create the runDir
+		if err = os.MkdirAll(runDir, os.ModeDir); err != nil {
+			fatal("Unable to create '%s'", runDir)
+			terminate()
+		}
+		warning("%s was not mounted, created directory and continuing", runDir)
```

Comment:
Rather than saying created directory and continuing - perhaps we should state the impact, i.e. it will not be possible to use `calicoctl node status` from the host.